src/aliasthis.c

```diff
@@ -69,6 +427,79 @@ Expression *resolveAliasThis(Scope *sc, Expression *e)
     return e;
 }
 
+/****************************************
+ * Expand alias this tuples.
```

Comment:
These two functions are taken from current implementation (from [expression.c](https://github.com/D-Programming-Language/dmd/blob/master/src/expression.c#L1017)) and they are used if type contains a tuple alias this.
I do not quite understand this case and I have disallowed multiple alias this if one of the symbols is tuple. Now it works as early.
Let's ask about these functions from their author.